slime/templates/cronjob.yaml

```diff
@@ -0,0 +1,295 @@
+          {{- end }}
+          {{- with .Values.cronJob.jobTemplate.hostAliases }}
+          hostAliases:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          {{- with .Values.cronJob.jobTemplate.hostIPC }}
+          hostIPC: {{ . }}
+          {{- end }}
+          {{- with .Values.cronJob.jobTemplate.hostPID }}
+          hostPID: {{ . }}
+          {{- end }}
+          {{- with .Values.cronJob.jobTemplate.hostUsers }}
+          hostUsers: {{ . }}
+          {{- end }}
+          {{- with .Values.cronJob.jobTemplate.hostname }}
+          hostname: {{ . }}
+          {{- end }}
+          {{- with .Values.cronJob.jobTemplate.imagePullSecrets }}
+          imagePullSecrets:
+            {{- tpl (. | toYaml) $root | nindent 12 }}
+          {{- end }}
+          {{- if .Values.initContainers.enabled }}
+          initContainers:
+          {{- range $container := .Values.cronJob.jobTemplate.initContainers.containers }}
+            - name: {{ include "slime.fullname" $root }}-{{ $container.name }}
+              image: {{ $container.image.repository }}:{{ $container.image.tag }}
+              {{- with $container.image.pullPolicy }}
+              imagePullPolicy: {{ . }}
+              {{- end }}
+              {{- with $container.command }}
+              command:
+                {{- toYaml . | nindent 14 }}
+              {{- end }}
+              {{- with $container.args }}
+              args:
+                {{- toYaml . | nindent 14 }}
+              {{- end }}
+              {{- with $container.workingDir }}
+              workingDir: {{ . }}
+              {{- end }}
+              {{- with $container.env }}
+              env:
+                {{- tpl (. | toYaml) $root | nindent 16 }}
+              {{- end }}
+              {{- with $container.envFrom }}
+              envFrom:
+                {{- tpl (. | toYaml) $root | nindent 14 }}
+              {{- end }}
+              {{- with $container.volumeMounts }}
+              volumeMounts:
+                {{- tpl (. | toYaml) $root | nindent 14 }}
+              {{- end }}
+          {{- end }}
+          {{- end }}
+          {{- with .Values.cronJob.jobTemplate.nodeName }}
+          nodeName: {{ . }}
+          {{- end }}
+          {{- with .Values.cronJob.jobTemplate.nodeSelector }}
+          nodeSelector:
+            {{- tpl (. | toYaml) $root | nindent 12 }}
+          {{- end }}
+          {{- with .Values.cronJob.jobTemplate.preemptionPolicy }}
+          preemptionPolicy: {{ . }}
+          {{- end }}
+          {{- with .Values.cronJob.jobTemplate.priority }}
+          priority: {{ . }}
+          {{- end }}
+          {{- with .Values.cronJob.jobTemplate.priorityClassName }}
+          priorityClassName: {{ . }}
+          {{- end }}
+          {{- with .Values.cronJob.jobTemplate.readinessGates }}
+          readinessGates:
+            {{- tpl (. | toYaml) $root | nindent 12 }}
+          {{- end }}
```

Comment:
The parameter is in the PodSpec, but it is not necessary in the JobTemplate.

Reply:
I removed the readinessGates in this commit.

https://github.com/chatwork/charts/pull/286/commits/98bc824ee09449577f9262dea2f80f13f93ef89b